spec/support/api/v3/shared_available_principals_examples.rb

```diff
@@ -104,6 +114,11 @@
 
     before { get href }
 
-    it_behaves_like 'unauthorized access'
+    if work_package_scope
+      it_behaves_like 'not found',
+                      I18n.t('api_v3.errors.not_found.work_package')
```

Comment:
Default behavior for work-package-scoped endpoints is not to reveal the presence of restricted work packages